modules/rostests/kmtests/ntos_ke/KeSpinLock.c

```diff
@@ -398,7 +430,37 @@ START_TEST(KeSpinLock)
     if (!KmtIsMultiProcessorBuild && !KmtIsCheckedBuild)
         pSpinLock = NULL;
 
-    for (i = 0; i < sizeof TestData / sizeof TestData[0]; ++i)
+    switch (GetCurrentNTVersion())
+    {
+#ifdef _M_X64
+        case _WIN32_WINNT_WS03:
+            TestData = TestDataWS03;
+            TestDataSize = sizeof(TestDataWS03);
```

Comment:
Could we instead use a `TestDataCount = _countof(TestDataWS03)` etc. , and use that in the for-loop:
`for (i = 0; i < TestDataCount; ++i)`  ?